TestingLowerBounds/Convex.lean

```diff
@@ -59,3 +61,20 @@ lemma ConvexOn.slope_tendsto_atTop (hf_cvx : ConvexOn ℝ (Set.Ici 0) f) :
   · simpa using h_mono 1 y zero_lt_one (not_le.mp hy).le
   · exact absurd (hxy.trans hy) hx
   · simpa using h_mono x y (zero_lt_one.trans (not_le.mp hx)) hxy
+
+lemma comp_neg {𝕜 F β : Type*} [LinearOrderedField 𝕜] [AddCommGroup F]
+    [OrderedAddCommMonoid β] [Module 𝕜 F] [SMul 𝕜 β] {f : F → β} {s : Set F}
+    (hf : ConvexOn 𝕜 s f) :
+    ConvexOn 𝕜 (-s) (f ∘ Neg.neg) := by
```

Comment:
Lemmas of that type are most often stated with the function in the form `fun x ↦ f (-x)` rather than `f ∘ Neg.neg`, because the first appears more naturally in code where you would want to use it.
Can you try that change here and elsewhere in the PR?